### PR TITLE
Dockerization test

### PR DESCRIPTION
**Pull request by alexey-gruzdev**
* Adding test for *workspace dockerize* command
* Renaming tests folder

### CODE REVIEW

File: openfl-docker/Dockerfile.base [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # If your machine is behind a proxy, make sure you set it up in ~/.docker/config.json
 
-ARG BASE_IMAGE=python:3.6.12-slim-stretch
+ARG BASE_IMAGE=python:3.8
```
Comment (tonyreina) [inline openfl-docker/Dockerfile.base:6]: Excellent. Glad to see the update to 3.8

**Review by tonyreina — APPROVED** [review]
Looks good.

**Comment by itrushkin** [comment]
@tonyreina @psfoley There is a couple of changes to CI required. I am working on it now, and you can merge this PR as soon as Jenkins build succeeds.

**Comment by itrushkin** [comment]
There was an error during `dockerization_test.sh` execution:
```
EXCEPTION : load_pem_x509_csr() missing 1 required positional argument: 'backend'
```
[Jenkins job with execution log](http://213.221.44.203/job/Federated-Learning/job/docker/51/console)
Probably there was an update of `cryptography` pip package API that leads to this. I am investigating this problem.

**Comment by itrushkin** [comment]
`cryptography<3` has a different API. This version gets installed with `python3-pip` on `ubuntu:18.04` docker image that is used for CI. I will specify the version manually but later we might want to pin it to the latest one.

**Comment by alexey-gruzdev** [comment]
> `cryptography<3` has a different API. This version gets installed with `python3-pip` on `ubuntu:18.04` docker image that is used for CI. I will specify the version manually but later we might want to pin it to the latest one.

I am pinning version of cryptography to 3.4.6 to avoid this issue

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Pull request merged**